README.md

```diff
@@ -387,10 +391,10 @@ In addition, there are a few significant changes to note:
   the source manager with (e.g. an overridden `YoutubeTrackDetailsLoader`), this **is not** compatible
   with this source manager.
 
-- Support for logging into accounts as a means of playing age-restricted tracks has been removed, with the
-  `TVHTML5EMBEDDED` client instead being the preferred workaround. There were a large number of
-  reasons for this change, but not least the fact that logging in was slowly becoming problematic and deprecated
-  on the YouTube backend. The amount of code to support this feature meant that it has been axed.
+- Support for logging into accounts as a means of playing age-restricted tracks has been removed.
```

Comment:
This entire section should actually be removed now that OAuth integration exists.